gears/flywheel/ants-registrationsynquick.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "ants-registrationsynquick",
+  "label": "Ants_RegistrationSynQuick using Nipype",
+  "description": "A gear to perform ANTS registration using Nipype and RegistrationSynQuick",
+  "author": "Flywheel",
+  "cite": "",
+  "maintainer": "support@flywheel.io",
+  "source": "https://gitlab.com/flywheel-io/flywheel-apps/ants-registrationsynquick",
+  "url": "https://nipype.readthedocs.io/en/latest/api/generated/nipype.interfaces.ants.registration.html#registrationsynquick",
+  "version": "0.1.2_2.3.5",
+  "custom": {
+    "flywheel": {
+      "suite": "NIPYPE"
+    },
+    "gear-builder": {
+      "category": "analysis",
+      "image": "flywheel/ants-registrationsynquick:0.1.2_2.3.5"
+    }
+  },
+  "license": "Other",
+  "inputs": {
+    "api_key": {
+      "base": "api-key"
+    },
+    "moving_image": {
+      "description": "Image to apply transformation to (generally a coregistered functional). Maps to a command-line argument: %s.",
+      "base": "file",
+      "optional": false,
```

Comment:
i seem to remember some weird quirk if the `optional` is added when it's actually. I think I just omit this if it's required. If you haven't run into any issues with local testing or testing on GA, then it's fine